### ISSUE TITLE
Catapult Android and Windows Tryservers offline

### ISSUE THREAD

**Issue by petrcermak** [catapult-project/catapult#2866]
https://build.chromium.org/p/tryserver.client.catapult/console

@anniesullie, @nedn, @eakuefner

**Comment by anniesullie** [comment]
I think this is crbug.com/650141

**Comment by petrcermak** [comment]
They are both online now, but [Catapult Windows Tryserver](https://build.chromium.org/p/tryserver.client.catapult/builders/Catapult%20Windows%20Tryserver/) fails in the [Dashboard Dev Server Tests Canary step](https://build.chromium.org/p/tryserver.client.catapult/builders/Catapult%20Windows%20Tryserver/builds/4891/steps/Dashboard%20Dev%20Server%20Tests%20Canary)

**Comment by anniesullie** [comment]
The next build succeeded, maybe it is a fluke?

**Comment by petrcermak** [comment]
Yeah. Note that there were at least 3 such flakes in the [last 20 builds](https://build.chromium.org/p/tryserver.client.catapult/builders/Catapult%20Windows%20Tryserver/):

* https://build.chromium.org/p/tryserver.client.catapult/builders/Catapult%20Windows%20Tryserver/builds/4891
* https://build.chromium.org/p/tryserver.client.catapult/builders/Catapult%20Windows%20Tryserver/builds/4888
* https://build.chromium.org/p/tryserver.client.catapult/builders/Catapult%20Windows%20Tryserver/builds/4887

**Comment by anniesullie** [comment]
Updated the title to reflect the current problem. @Apeliotes can you take a look? Is there someplace we could add more logging?

**Comment by petrcermak** [comment]
Another flake: https://build.chromium.org/p/tryserver.client.catapult/builders/Catapult%20Windows%20Tryserver/builds/4908/steps/Dashboard%20Dev%20Server%20Tests%20Canary/logs/stdio

**Comment by petrcermak** [comment]
Another failure (with some __potentially useful output__, https://build.chromium.org/p/tryserver.client.catapult/builders/Catapult%20Windows%20Tryserver/builds/4909/steps/Tracing%20Dev%20Server%20Tests%20Canary/logs/stdio):

```
[0927/082801:ERROR:process_info.cc(608)] range at 0x6d50256200000000, size 0x5e fully unreadable
[0927/082801:ERROR:process_info.cc(608)] range at 0x6d50258200000000, size 0x5e fully unreadable
[0927/082801:ERROR:process_info.cc(608)] range at 0x0, size 0x5e fully unreadable
```
